src/pages/commits/constants.ts

```diff
@@ -1,4 +1,21 @@
 const commitChartHeight = 224;
 const gridHeight = 226;
 
-export { commitChartHeight, gridHeight };
+const TASK_ICONS_PER_ROW = 6;
+const TASK_ICON_HEIGHT = 24;
+const TASK_ICON_PADDING = 16;
+
+const GROUPED_BADGE_PER_ROW = 2;
+const GROUPED_BADGE_HEIGHT = 40;
+const GROUPED_BADGE_PADDING = 8;
```

Comment:
Where do these values come from?

Reply:
The `TASK_ICONS_PER_ROW` and `GROUPED_BADGE_PER_ROW` are more observational values. From the image below you can see that at most there will be 6 task icons per row, and 2 grouped badges per row:

<img width="200" alt="Screen Shot 2022-04-08 at 11 22 44 AM" src="https://user-images.githubusercontent.com/47064971/162479970-385c9ceb-ac75-44e9-93c0-331fd3b0dd55.png">

The padding and height correspond to the actual padding/heights of the task icons and grouped badges. To make it clearer, I altered the styles of these components so that these constants are used when appropriate 😇